lib/src/_shared/PickerToolbar.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
 import clsx from 'clsx';
 import Toolbar, { ToolbarProps } from '@material-ui/core/Toolbar';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
```

Comment:
To investigate if we can't get rid of this module. However, without the bundle size tracking tool of the core repository, it might be a bit challenging to measure progress on the impact.